model/model_training/configs/zero3_config.json

```diff
@@ -12,12 +20,12 @@
     }
   },
   "scheduler": {
-    "type": "WarmupDecayLR",
+    "type": "WarmupLR",
```

Comment:
Can we create different zero config files for WarmupLR and WarmupDecayLR? For the last runs I changed it manually but I think it would be better to have pre-training (which would refer the WarmupLR zero config) + fine-tuning (using WarmupDecalyLR zero config) .. this will make it simpler/possible to reproduce our results.